larpandoracontent/LArHelpers/LArMvaHelper.h

```diff
@@ -1,26 +1,57 @@
 /**
- *  @file   larpandoracontent/include/LArHelpers/LArSvmHelper.h
+ *  @file   larpandoracontent/include/LArHelpers/LArMvaHelper.h
  *
- *  @brief  Header file for the lar svm helper class.
+ *  @brief  Header file for the lar mva helper class.
  *
  *  $Log: $
  */
-#ifndef LAR_SVM_HELPER_H
-#define LAR_SVM_HELPER_H 1
-
-#include "larpandoracontent/LArObjects/LArSupportVectorMachine.h"
+#ifndef LAR_MVA_HELPER_H
+#define LAR_MVA_HELPER_H 1
 
 #include <fstream>
 #include <chrono>
 #include <ctime>
 
+class AdaBoostDecisionTree;
+class SupportVectorMachine;
+
 namespace lar_content
 {
 
+typedef std::vector<double> DoubleVector; 
```

Comment:
Would prefer `DoubleVector` to be public, within `MvaFeatureTool` or `LArMvaHelper` classes. Would control use of this typedef.

Reply:
I left this here so that when you add DoubleVector to the pandora typedefs you wouldn't have to change any code currently using DoubleVector.  I was under the impression that this was the plan, but if you don't want to add DoubleVector to the pandora typedefs I'll change the location of this to make sure it's use is policed.